temp-legacy-code/src/main/java/com/ivy/legacy/datamodel/Tag.kt

```diff
@@ -0,0 +1,39 @@
+package com.ivy.legacy.datamodel
+
+import com.ivy.data.db.entity.TagEntity
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Tag(
+    val id: UUID = UUID.randomUUID(),
+
+    val name: String,
+    val description: String? = null,
```

Comment:
Let's remove default values to prevent cases where some contributors construct the class w/o mapping all fields

Reply:
Sure